contracts/account/proxy/src/commands.rs

```diff
@@ -25,6 +25,23 @@ pub fn execute_module_action(
     Ok(ProxyResponse::action("execute_module_action").add_messages(msgs))
 }
 
+/// Executes actions forwarded by whitelisted contracts
+/// This contracts acts as a proxy contract for the dApps
+pub fn execute_module_action_response(
```

Comment:
We will need to add an API for execute_with_action_response ? Another AcountAction like variant ?

Reply:
I don't think so, relying on the data in a `Response` is a very specific use-case which is very rare (i've never done it apart from instantiations to get the addr). So with the `execute_with_data` the developer want to do a single, very specific action that returns some data that they want to recover. I don't think we need to create a new type to incapsulate that behavior because I don't think there a good way to do it, or a real need for it.